RefactoringReport.md

```
@@ -0,0 +1,60 @@
+    - `useRestoreManualBackup.ts`: Takes `onError` but doesn't use `useRef`/`useEffect` pattern.
+    - `useCreateManualBackup.ts`: Takes `onError` but doesn't use `useRef`/`useEffect` pattern.
+
+- **Page-specific Hooks:**
+    - `cat-launcher/src/pages/game-tips/hooks/useGetTips.ts`: Missing error callbacks and `useEffect` sync.
+    - `cat-launcher/src/pages/TilesetsPage/hooks.ts`: `useGetThirdPartyTilesetInstallationStatus` and `useListAllTilesets` missing callbacks. `useUninstallThirdPartyTileset` doesn't use `useRef`/`useEffect`.
+    - `cat-launcher/src/pages/ModsPage/hooks/useUninstallThirdPartyMod.ts`: Doesn't use `useRef`/`useEffect`.
+    - `cat-launcher/src/pages/ModsPage/hooks/useGetThirdPartyModInstallationStatus.ts`: Missing error callbacks.
+    - `cat-launcher/src/pages/PlayPage/hooks/useLaunchGame.ts`: Takes `onError` but uses it directly in `useMutation` without `useRef`.
+    - `cat-launcher/src/pages/SoundpacksPage/hooks.ts`: `useGetThirdPartySoundpackInstallationStatus` and `useListAllSoundpacks` missing callbacks. `useUninstallThirdPartySoundpack` doesn't use `useRef`/`useEffect`.
+
+### Suggested Fixes:
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)